.gitignore

```
@@ -1,11 +1,8 @@
 # created by virtualenv automatically
 .idea/
-.flake8
-src/__init__.py
 src/__pycache__/
```

Правильно так:
```suggestion
__pycache__/
```
Тогда ты исключишь вообще все вхождения этой директории

В репозиторий попала директория `src/__pycache__/` которую нужно удалить